src/components/utrecht/Select.tsx

```diff
@@ -0,0 +1,68 @@
+import clsx from "clsx";
+import { ForwardedRef, forwardRef, PropsWithChildren, SelectHTMLAttributes } from "react";
+
+interface SelectProps extends SelectHTMLAttributes<HTMLSelectElement> {
+  invalid?: boolean;
+}
+
+export const Select = forwardRef(
+  (
+    { disabled, invalid, required, className, children, ...restProps }: PropsWithChildren<SelectProps>,
+    ref: ForwardedRef<HTMLSelectElement>
+  ) => {
+    const noscript = true;
+    return (
+      <select
+        {...restProps}
+        ref={ref}
+        aria-invalid={invalid || undefined}
+        disabled={disabled}
+        required={noscript ? required : false}
+        aria-required={noscript ? undefined : required} // Don't let it affect CSS :invalid
+        className={clsx(
+          "utrecht-select",
+          disabled && "utrecht-select--disabled",
+          invalid && "utrecht-select--invalid",
+          required && "utrecht-select--required",
+          className
+        )}
+      >
+        {children}
+      </select>
+    );
+  }
+);
+
+Select.displayName = "utrecht-radio-button";
+
+interface SelectOptionProps extends SelectHTMLAttributes<HTMLOptionElement> {
```

Comment:
Should this be SelectHTMLAttributes when it's an option element?

Reply:
Will fix this to use `OptionHTMLAttributes`